pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since we are going to host other projects e.g. the data service written in JS, I'd suggest to move the whole project structure in a dedicated directory.

Reply:
Changed the structure and updated the README